stacker_blueprints/dynamodb.py

```diff
@@ -0,0 +1,152 @@
+from stacker.blueprints.base import Blueprint
+from . import util
+
+from troposphere import (
+    dynamodb2,
+    Ref,
+    GetAtt,
+    Output,
+)
+
+from troposphere.route53 import RecordSetType
+
+def prep_schemata(config):
+    try:
```

Comment:
It's unlikely that this would happen, but in general I try to keep try/except blocks as small as they have to be, so they cover the smallest # of use cases.  In this case, the `IndexError` is only really expected to come from `config["KeySchema"]`, right?  So I'd probably change this to:

```
try:
    schemas = config["KeySchema"]
except IndexError:
    raise IndexError(...)

for schema in schemas:
    ...
```